### PR TITLE
Calendar stylings and html tags

### PR DESCRIPTION
**Pull request by drewvolz**
This makes the view have less html and more consistency.

### CODE REVIEW

File: views/calendar/event.js [hunk 1]
```diff
@@ -53,8 +53,15 @@ function getString(date) {
   return `${month}/${day} ${hours}:${min}`
 }
 
+function stripAndClean(text: string) {
```
Comment (hawkrives) [inline views/calendar/event.js]: What happened to HTMLparser?

**Comment by drewvolz** [comment]
If you are referring to this

```
let Entities = require('html-entities').AllHtmlEntities
const entities = new Entities()
```

...then it won't work. It does not strip out the `<i></i>` tags.

**Comment by hawkrives** [comment]
I'm referring to the parseHtml/getText combo from /lib that I mentioned earlier.

**Comment by drewvolz** [comment]
I missed that completely. I'll make the changes happen with that, then. Thanks.

**Comment by hawkrives** [comment]
I'm confused by one thing. 

We've got two PRs that need to do the same thing, yet the HTML entities decoding only works to remove the tags in one case?

**Comment by drewvolz** [comment]
They both can probably be solved by the HTML parser you have in lib. That was my misunderstanding.

File: views/calendar/calendar.js [hunk 2]
```diff
@@ -144,6 +144,7 @@ let styles = StyleSheet.create({
     backgroundColor: '#ffffff',
   },
   row: {
+    minHeight: 88,
```
Comment (hawkrives) [inline views/calendar/calendar.js:147]: What does `minHeight` do to the title in the one-line case? Does the title stay at the top of the row?

**Comment by drewvolz** [comment]
It keeps it at the top, correct.
screen shot 2016-08-20 at 10 56 40 pm

**Comment by hawkrives** [comment]
LGTM

**Pull request merged**